instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_sqs.py

```diff
@@ -0,0 +1,66 @@
+import botocore.session
+from moto import mock_sqs
+
+from opentelemetry.instrumentation.botocore import BotocoreInstrumentor
+from opentelemetry.semconv.trace import SpanAttributes
+from opentelemetry.test.test_base import TestBase
+
+
+class TestDynamoDbExtension(TestBase):
+    def setUp(self):
+        super().setUp()
+        BotocoreInstrumentor().instrument()
+
+        session = botocore.session.get_session()
+        session.set_credentials(
+            access_key="access-key", secret_key="secret-key"
+        )
+        self.region = "us-west-2"
+        self.client = session.create_client("sqs", region_name=self.region)
+
+    def tearDown(self):
+        super().tearDown()
+        BotocoreInstrumentor().uninstrument()
+
+    @mock_sqs
+    def test_sqs_messaging_send_message(self):
+        create_queue_result = self.client.create_queue(QueueName="test_queue_name")
+        response = self.client.send_message(QueueUrl=create_queue_result["QueueUrl"], MessageBody="content")
+
+        spans = self.memory_exporter.get_finished_spans()
+        assert spans
+        self.assertEqual(len(spans), 2)
+        span = spans[1]
+        self.assertEqual(span.attributes[SpanAttributes.MESSAGING_SYSTEM], "aws.sqs")
```

Comment:
[This is not the ](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md#messaging-attributes) says the value should be `AmazonSQS`.

Reply:
This is what they did in node:
https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/sqs.ts#L160
and in semconv:
<img width="410" alt="image" src="https://user-images.githubusercontent.com/47108628/191689926-fdb7619b-81a9-4b7e-83e3-f535e407bb59.png">
<img width="668" alt="image" src="https://user-images.githubusercontent.com/47108628/191689976-69194ae4-4c02-4df7-b35c-c83d7bbaf3ed.png">